client/src/components/Header/index.style.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import {Link} from 'react-router-dom';
+export const StyledHeader = styled.div`
+display: flex;
+flex-direction:column;
+width:20%;
+z-index: -1;
+top: 0;
+right: 0;
+position:fixed;
```

Comment:
This css is very complicated!! Please simplify it, you don't have to use floats.